definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -120,7 +120,15 @@ declare class moment$Moment {
   static ISO_8601: string;
   static (): moment$Moment;
   static (
-    initDate: moment$MomentOptions | number | Date | Array<number> | moment$Moment | string
+    initDate:
+      | moment$MomentOptions
+      | number
+      | Date
+      | Array<number>
+      | moment$Moment
+      | string
+      | null
+      | void
```

Comment:
I could've used `?moment$MomentOptions` but I felt like it was more clear to explicitly mark `null` and `void` in this type. What do you think?